src/mol-io/writer/cif/encoder.ts

```diff
@@ -324,6 +324,9 @@ function cifFieldsFromTableSchema(schema: Table.Schema) {
             fields.push({ name: k, type: Field.Type.Str, value: columnListValue(k), valueKind: columnValueKind(k) });
         } else if (t.valueType === 'tensor') {
             fields.push(...getTensorDefinitions(k, t.space));
+        } else if (t.valueType === 'any') {
+            // NOTE: test property is the "str" type will work here
```

Comment:
I don't understand

Reply:
`Is => if` typo. But will be removing this code.